.github/workflows/generate_README.yaml

```diff
@@ -0,0 +1,52 @@
+name: Generate README
+on:
+  workflow_call:
+    inputs:
+      BRANCH_NAME:
+        required: true
+        type: string
+
+jobs:
+  make-README:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.BRANCH_NAME }}
+      - name: Fill template
+        working-directory: "docs/"
+        timeout-minutes: 0.1
+        run: |
+          sub_doc_parts_from() {
```

Comment:
I would put this in a script in a releng directory as it's easier to test

Reply:
Done